src/aznfswatchdog

```diff
@@ -122,22 +149,57 @@ reconcile_conntrack()
             if [[ "$line" =~ $matchstr ]]; then
                 l_sport=${BASH_REMATCH[1]}
                 l_dport=${BASH_REMATCH[2]}
-                reconcile_conntrack_one $l_ip $l_sport $l_dport $l_nfsip
+                reconcile_conntrack_one $l_ip $l_sport $l_dport $l_nfsip "SYN_SENT"
+            fi
+        done
+    fi
+
+    IFS=$'\n' output111=$(conntrack -L -p tcp -d $l_ip --dport 111 --state ESTABLISHED 2>/dev/null)
+    IFS=$'\n' output2048=$(conntrack -L -p tcp -d $l_ip --dport 2048 --state ESTABLISHED 2>/dev/null)
```

Comment:
filter UNREPLIED entries only and don't restrict to ESTABLISHED state only. Sometimes if there is some n/w issue it may be in some other state. What we want to clear are entries with dst ip equal to the proxy IP and ports matching 111 and 2048. if such an entry is stuck in UNREPLIED state for more than 30 secs we want to delete it.

    IFS=$'\n' output111=$(conntrack -L -p tcp -d $l_ip --dport 111 | grep "\[UNREPLIED\]" 2>/dev/null)

    IFS=$'\n' output2048=$(conntrack -L -p tcp -d $l_ip --dport 2048 | grep "\[UNREPLIED\]" 2>/dev/null)